pkg/columns/columns.go

```diff
@@ -624,6 +624,28 @@ func GetFieldAsStringExt[T any](column ColumnInternals, floatFormat byte, floatP
 			}
 		}
 
+		return func(entry *T) string {
+			return "TODO"
+		}
+	case reflect.Map:
+		keyType := column.(*Column[T]).Type().Key()
+		valueType := column.(*Column[T]).Type().Elem()
+
+		if keyType.Kind() == reflect.String && valueType.Kind() == reflect.String {
+			ff := GetFieldFunc[map[string]string, T](column)
+			return func(entry *T) string {
+				result := strings.Builder{}
+				m := ff(entry)
+				for k, v := range m {
+					result.WriteString(k)
+					result.WriteString("=")
+					result.WriteString(v)
+					result.WriteString(",")
+				}
```

Comment:
Ordering so that the result is deterministic:

```suggestion
				m := ff(entry)
				kvPairs := []string
				for k, v := range m {
					kvPairs = append(kvPairs, fmt.Sprintf("%s=%s", k, v)
				}
				sort.Strings(kvPairs)
				for _, kvPair := range kvPairs {
					result.WriteString(kvPair)
				}

```